app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -115,9 +115,9 @@ def redirect_to_new_continuous_applications_if_eligible
 
     def redirect_to_application_if_signed_in
       if candidate_signed_in?
-        return redirect_to_new_continuous_applications_if_eligible if current_application.continuous_applications?
+        return redirect_to candidate_interface_application_complete_path if current_application.v23?
 
-        redirect_to candidate_interface_application_complete_path
+        redirect_to_new_continuous_applications_if_eligible
```

Comment:
What happen if you comment it out `redirect_to_new_continuous_applications_if_eligible` or `return redirect_to candidate_interface_application_complete_path if current_application.v23?`.
What specs fail? 
*Just double checking that we have the necessary coverage before merging.

Reply:
Perfect, thanks! I think then is good to change.